src/PKSim.Core/PKSimRegistrationConvention.cs

```diff
@@ -24,6 +25,9 @@ public override void Process(Type concreteType, IContainer container, LifeStyle
          if (concreteType.IsNested)
             return;
 
+         if (concreteType.GetCustomAttribute<System.Runtime.CompilerServices.CompilerGeneratedAttribute>() != null)
+            return;
```

Comment:
Not sure how this broke, but I found that some compiler generated classes were registered more than once breaking this scanning convention.

Reply:
There is a transitive update to Castle Core involved here.